### PR TITLE
WIP honor branding in authentication

### PR DESCRIPTION
**Pull request by sallyom**
bug 1700891
opening PR to experiment

### CODE REVIEW

File: pkg/operator2/managed_console_config.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+func (c *authOperator) handleManagedConfigMap() *corev1.ConfigMap {
+	managedConfig, err := c.configMaps.ConfigMaps(managedConfigNamespace).Get(managedConsoleConfig, metav1.GetOptions{})
+	if err != nil && !kerrors.IsNotFound(err) {
+		klog.Infof("error getting managed config: %v", err)
+		return nil
```
Comment (enj) [inline pkg/operator2/managed_console_config.go]: You need to error out or provide a safe default.

File: pkg/operator2/oauth.go [hunk 2]
```diff
@@ -150,10 +151,11 @@ func (c *authOperator) handleOAuthConfig(
 	}
 
 	cliConfigBytes := encodeOrDie(cliConfig)
+	newConsoleConfig := managedConsoleConfigBytes(managedConsoleConfig)
 
-	completeConfigBytes, err := resourcemerge.MergeProcessConfig(nil, cliConfigBytes, operatorConfig.Spec.UnsupportedConfigOverrides.Raw)
+	completeConfigBytes, err := resourcemerge.MergeProcessConfig(nil, cliConfigBytes, newConsoleConfig, operatorConfig.Spec.UnsupportedConfigOverrides.Raw)
```
Comment (enj) [inline pkg/operator2/oauth.go]: You cannot merge the branding config map on top of the OAuth server config.

Comment (sallyom) [reply pkg/operator2/oauth.go]: yea figured that-but wasn't sure how the overriding of console config was going to happen so wanted to run this to see (notice the WIP and /hold) : )

**Comment by sallyom** [comment]
closing in favor of https://github.com/openshift/cluster-authentication-operator/pull/121

**Pull request closed without merging**